packages/ndla-ui/src/locale/messages-en.ts

```diff
@@ -1167,6 +1167,18 @@ const messages = {
         },
       },
     },
+    arena: {
+      title: 'Arena',
+      notification: {
+        title: 'Notifications',
+        showAll: 'View all notifications',
+        description:
+          'Welcome to NDLAs Arena. Here you can discuss, share and cooperate with other teachers from the entire country. You will find forums for the different subjects and themes.',
```

Comment:
Tenker det er bedre uten eigendoms-s også på engelsk. 
```suggestion
          'Welcome to NDLA Arena. Here you can discuss, share and cooperate with other teachers from all of Norway. You will find forums for the different subjects and themes.',
```